smalt/0.7.6/readme.md

```diff
@@ -0,0 +1,35 @@
+# Smalt Container
+
+## Documentation and Overview
+
+Main Tool: [SMALT – A new mapper for DNA sequencing reads](https://www.researchgate.net/publication/266155341_SMALT_-_A_new_mapper_for_DNA_sequencing_reads)
+
+Full documentation: https://www.sanger.ac.uk/tool/smalt-0/ 
+
+SMALT aligns DNA sequencing reads with a reference genome.
+
+This cointainer also contains samtools=1.15 as this was needed for a particular workflow. Also, bambamc=0.0.50 was added which allows passing of bam files to smalt see [documentation](https://github.com/rcallahan/smalt).
+
+```
+              SMALT - Sequence Mapping and Alignment Tool
+                             (version: 0.7.6)
+SYNOPSIS:
+    smalt <task> [TASK_OPTIONS] [<index_name> <file_name_A> [<file_name_B>]]
+
+Available tasks:
+    smalt check   - checks FASTA/FASTQ input
+    smalt help    - prints a brief summary of this software
+    smalt index   - builds an index of k-mer words for the reference
+    smalt map     - maps single or paired reads onto the reference
+    smalt sample  - sample insert sizes for paired reads
+    smalt version - prints version information
+
+Help on individual tasks:
+    smalt <task> -H
+```
+
+## Example Usage
+
+```
+smalt index -k 13 -s 6 PhiX174 GCF_000819615.1_ViralProj14015_genomic.fastq
+```
```

Comment:
Could you add what [PhiX174.sma](https://github.com/jvhagey/docker-builds/blob/patch-1/smalt/0.7.6/tests/PhiX174.sma) and [PhiX174.smi](https://github.com/jvhagey/docker-builds/blob/patch-1/smalt/0.7.6/tests/PhiX174.smi) are for? They are binaries, so it's difficult for me to evaluate them. And I didn't find anything after a quick glance at https://github.com/rcallahan/smalt

Reply:
These were just for [the test](https://github.com/jvhagey/docker-builds/blob/2e013bbd24a21b9826bbcde5ad2e965a67c8e8a0/smalt/0.7.6/Dockerfile#L47) they aren't needed to run smalt. I modelled it after [the freyja container](https://github.com/StaPH-B/docker-builds/blob/570ab110035c9447a030936123954f1977336b72/freyja/1.3.4/Dockerfile#L35). This is my first time working with github tests so if this isn't a great way to do it I am open to something else.

Reply:
kk. So they are results from running the test before and you are comparing the two files. That makes sense.

I'm not confused anymore.